src/processors/models/processorsManager.js

```diff
@@ -68,6 +68,23 @@ module.exports.redownloadJSProcessor = async function(processorId) {
     }
 };
 
+module.exports.updateProcessor = async function (processorId, processor) {
+    const oldProcessor = await databaseConnector.getProcessor(processorId);
+    if (!oldProcessor) {
+        throw generateProcessorNotFoundError();
+    }
+    processor.created_at = oldProcessor.created_at;
+    if (processor.type === 'file_download') {
+        let newJavascript = await fileManager.downloadFile(processor.file_url);
+        fileManager.validateJavascriptContent(newJavascript);
+        processor.javascript = newJavascript;
+    } else {
```

Comment:
not needed, if `processor.type !== constants.PROCESSOR_TYPE_FILE_DOWNLOAD` then file_url should be undefined

Reply:
done